chainer/backends/_chainerx.py

```diff
@@ -165,11 +164,7 @@ def _array_from_chainerx(array):
     if array is None:
         return None
     if not isinstance(array, chainerx.ndarray):
-        if isinstance(array, chainer.get_array_types()):
-            return array
-        raise TypeError(
-            'Tried to convert to a non-ChainerX array from an invalid type: '
-            '{}'.format(type(array)))
+        return array
```

Comment:
I see this behavior is convenient (for slices?) but it'll stop being symmetrical to "to_chainerx". Is it difficult to keep the old behavior and check the type before calling this function?